src/strategies/math/README.md

```diff
@@ -2,25 +2,78 @@
 
 Apply common mathematical operations on outputs from other strategies.
 
+## Operations
+
 Currently supported operations are:
 
-- `square-root`
-- `cube-root`
+| Operation     | Operand Count | Description                                |
+| ------------- | ------------- | ------------------------------------------ |
+| `square-root` | 1             | takes the square root of the operand       |
+| `cube-root`   | 1             | takes the cube root of the operand         |
+| `min`         | 2             | takes the smaller number of the 2 operands |
+| `max`         | 2             | takes the larger number of the 2 operands  |
+
+## Examples
 
 The following example takes the cube root of a user's DAI token balance as voting score.
 
 ```json
 {
   "symbol": "MATH",
-  "strategy": {
-    "name": "erc20-balance-of",
-    "network": "1",
-    "params": {
-      "address": "0x6b175474e89094c44da98b954eedeac495271d0f",
-      "symbol": "DAI",
-      "decimals": 18
+  "operands": [
```

Comment:
You shouldn't change the structure of the params here it may break results for space currently using this strategy

Reply:
I understand the concern. However, the `math` strategy was created for composibility, and would be greatly limited in terms of what can be built on top of it if multi-operand support cannot be added.

The change in structure is backward-compatible, as the strategy automatically converts the old param format into the new one.

Also there doesn't seem to be any space using this strategy yet:

```sh
$ curl "https://hub.snapshot.org/api/explore" | jq ".strategies.math" null
```

Reply:
@bonustrack Any update on this? Thank you so much.

Reply:
Thanks make sense, apologies for slow review